src/nprintml/label/aggregator/pcap.py

```diff
@@ -44,17 +44,28 @@ def __call__(self, compress=False, sample_size=1):
         print('Attaching labels to nPrints')
 
         labels = self.load_label(self.label_csv)
+        labels = self.rewrite_labels(labels, npts)
+
         (npt_df, missing_labels, ogns, nns) = self.attach_label(npts, labels)
 
         print('  labels attached: missing labels for:', missing_labels)
         print('    missing labels caused samples to be dropped:', (ogns - nns))
 
         return npt_df
-
-    def merge_npt(self):
-        """Merge nPrint data from multiple output files."""
-        # Load nPrint data sets and determine largest set
-        print('Loading nprints')
```

Comment:
The method `merge_npt` now appears to be undefined.

I'm guessing this won't work as is?

Reply:
Fixed, got mauled somehow.